From 52ebc5a9961729c6392390e423b83f1a94b7dd12 Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Fri, 25 May 2018 18:25:24 -0300
Subject: [PATCH] updatehub-server: Remove mount command from udev rule
Organization: O.S. Systems Software LTDA.

Udev cannot mount this once UpdateHub server mounts device in
/mnt/updatehub.

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 cmd/updatehub-server/udev.rules | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmd/updatehub-server/udev.rules b/cmd/updatehub-server/udev.rules
index 7ab8580..3c2b75f 100644
--- a/cmd/updatehub-server/udev.rules
+++ b/cmd/updatehub-server/udev.rules
@@ -3,7 +3,6 @@ ENV{ID_FS_LABEL}=="updatehub|UPDATEHUB", \
   ACTION=="add", \
   SUBSYSTEM=="block", \
   RUN+="/bin/mkdir -p /mnt/updatehub", \
-  RUN+="/bin/mount -o ro /dev/%k /mnt/updatehub", \
   RUN+="/bin/sh -c '/bin/echo /usr/bin/updatehub-server --mount /dev/%k --fstype $env{ID_FS_TYPE} --wait /mnt/updatehub | at -M now'"
 
 ENV{ID_FS_LABEL}=="updatehub|UPDATEHUB", \
-- 
2.17.0

